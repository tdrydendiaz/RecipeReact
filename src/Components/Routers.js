import React, { Component } from 'react';
import { BrowserRouter as Router, Route, Link } from "react-router-dom";
import App from '../App'
import MoreDetail from './MoreDetail'

export default class Routers extends Component {

    render() {

        return (
           
            <p></p>

        );
  
}
}
 // <Router>
            //     <div>
            //         <ul>
            //             <li><Link to="/">Home</Link></li>
            //             <li><Link to="/topics">Add New Recipe</Link></li>
            //         </ul>
            //         <Route exact path="/" component={App} />
            //         <Route path="/MoreDetail" component={MoreDetail} />
            //     </div>
            // </Router>