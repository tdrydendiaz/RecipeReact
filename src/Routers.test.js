import React from "react";
import { shallow } from "enzyme";
import Routers from "./Components/Routers";

it("contains text", () => {
  const wrapper = shallow(<Routers />);
  expect(wrapper).toMatchSnapshot();
});
