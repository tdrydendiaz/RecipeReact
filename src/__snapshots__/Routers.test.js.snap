// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`contains text 1`] = `ShallowWrapper {}`;
